src/utils/arfs_builders/arfs_drive_builders.ts

```diff
@@ -210,6 +210,14 @@ export class ArFSPrivateDriveBuilder extends ArFSMetadataEntityBuilder<ArFSPriva
 	}
 }
 
+// A utility type to assist with fail-safe decryption of private entities
+export class EncryptedEntityID extends EntityID {
+	constructor() {
+		super(`${stubEntityID}`); // Unused after next line
+		this.entityId = ENCRYPTED_DATA_PLACEHOLDER;
```

Comment:
So any encrypted entity id will equal any other unrelated encrypted entity id? :thinking:

Reply:
Does that comparison happen somewhere?

Reply:
Similar to root folder this is just for display purposes for "fail safe decryption" of private entities for the list-all-drives command.